### DISCUSSION TITLE
能不能整个简单的 package 制作demo, 一个 “hello world”

### DISCUSSION THREAD

**Discussion by sogwms** [xmake-io/xmake#4902] [General]
就现在的 package 文档， 不友好

**Comment by star-hengxing** [comment]
大部分 package desc 都是脏活，没有最佳实践，请提出明确的需求，比如说制作哪种类型的包

参考：https://zhuanlan.zhihu.com/p/651108523

**Reply by sogwms** [reply]
摸索出来怎么整了. 
但还是建议弄些 "hello world" 完善下文档，现在的文档基本都是“字典型”的。简单的案例比如说项目里有部分代码库想抽出来做成 "package", 如何制作，一些注意事项(e.g. 是否必须要有个“package索引仓库”, 指定package导出的头文件的方式，add_versions的必要性, ...)

**Reply by waruqi** [reply]
https://xmake.io/#/zh-cn/package/remote_package?id=%e6%b7%bb%e5%8a%a0%e5%8c%85%e5%88%b0%e4%bb%93%e5%ba%93
